### PR TITLE
Auth cli doc

### PR DESCRIPTION
**Pull request by ohlookadollar**

### CODE REVIEW

File: authn-authz/binding.md [hunk 1]
```diff
@@ -76,45 +76,131 @@ $ tanzu auth --kubeconfig /tmp/pinniped_kubeconfig.yaml binding add --user usern
...
+-n, --namespace string Namespace
+-r, --role string Role
+-u, --user string User Name
+
+Global Flags:
+--kubeconfig string kubeconfig file
+```
+
+#### Object Has Been Modified Error:
+This error is a race condition caused by running the mutiple rbac cli actions at the same time.
```
Comment (julia-pu) [inline authn-authz/binding.md]: typo, should be `caused by running multiple rbac cli actions`

File: authn-authz/binding.md [hunk 2]
```diff
@@ -76,45 +76,131 @@ $ tanzu auth --kubeconfig /tmp/pinniped_kubeconfig.yaml binding add --user usern
...
+-n, --namespace string Namespace
+-r, --role string Role
+-u, --user string User Name
+
+Global Flags:
+--kubeconfig string kubeconfig file
+```
+
+#### Could Not Find Error:
+Removing a subject (user or group) from a role that does not exist will produce the following error:
```
Comment (julia-pu) [inline authn-authz/binding.md]: Is this supposed to be removing a subject from a role that exists, but doesn't have the subject bound to it, or a role that doesn't exist at all?

File: authn-authz/binding.md [hunk 3]
```diff
@@ -76,45 +76,131 @@ $ tanzu auth --kubeconfig /tmp/pinniped_kubeconfig.yaml binding add --user usern
...
+-g, --group string User Group
+-h, --help help for remove-binding
+-n, --namespace string Namespace
+-r, --role string Role
+-u, --user string User Name
+```
+
+### <a id="troubleshooting"></a> Troubleshooting
+
+1) To Get a list of permissions for a user or a group
```
Comment (julia-pu) [inline authn-authz/binding.md]: Tiny thing, but could the wording be consistent between this point and the next? So either both `To get ...` or `Get ...`

File: authn-authz/binding.md (hunk 2 above)
Comment (ohlookadollar) [reply authn-authz/binding.md]: the error message is true for both. I suppose i could make that clear

File: authn-authz/binding.md (hunk 1 above)
Comment (ohlookadollar) [reply authn-authz/binding.md]: fixed

**Review by julia-pu — APPROVED** [review]

**Pull request merged**